app/code/Magento/Ui/view/base/web/js/grid/editing/editor-view.js

```diff
@@ -28,12 +28,18 @@ define([
                 '<!-- /ko -->',
             rowTmpl:
                 '<!-- ko with: _editor -->' +
-                    '<!-- ko if: isActive($row()._rowIndex, true) -->' +
-                        '<!-- ko with: getRecord($row()._rowIndex, true) -->' +
-                            '<!-- ko template: rowTmpl --><!-- /ko -->' +
-                        '<!-- /ko -->' +
-                        '<!-- ko if: isSingleEditing && singleEditingButtons -->' +
-                            '<!-- ko template: rowButtonsTmpl --><!-- /ko -->' +
+                    '<!-- ko if: typeof $row() !== "undefined" -->' +
+                        '<!-- ko if: isActive($row()._rowIndex, true) -->' +
+                            '<!-- ko if: typeof $row() !== "undefined" -->' +
```

Comment:
I think we can remove this condition as we are already checking the same in line number 31.

Reply:
@engcom-Hotel I cannot remove the code that checks for undefined rows. This code is necessary to check each column for undefined values, as the $row() method will return undefined if no rows are found. If we remove this code, the $row() undefined issue will come back.